src/io/flutter/devtools/DevToolsManager.java

```diff
@@ -259,26 +260,47 @@ private void openBrowserImpl(String uri, String screen) {
       devToolsInstance.openBrowserAndConnect(uri, screen);
     }
     else {
-      @Nullable final OSProcessHandler handler =
-        isBazel(project) ? getProcessHandlerForBazel() : getProcessHandlerForPub();
+      if (isBazel(project)) {
```

Comment:
comment why we do this for Bazel only for now (non-bazel projects currently have a flutter daemon with an older version of devtools).

Reply:
one more thing. we also don't do any flutter version checking so this wouldn't be safe externally as stable probably doesn't support this daemon api.